main.py

```diff
@@ -115,32 +118,43 @@ def _restore_status(self, switches_status, interfaces_status):
                 error = ('Error while restoring interface status. The '
                          f'interface {interface_id} does not exist.')
                 raise KeyError(error)
-
+        # links
+        for link, state, in self.links_state.items():
+            try:
+                if state:
+                    self.links[link].enable()
+                else:
+                    self.links[link].disable()
+            except KeyError:
+                error = ('Error restoring link status.'
+                         f'The link {link} does not exist.')
+                raise KeyError(error)
         log.info('Network status restored.')
 
     def _load_network_status(self):
         """Load network status saved in storehouse."""
-        switches_status = {}
-        interfaces_status = {}
         status = self.storehouse.get_data()
         if status:
             switches = status.get('network_status')['switches']
+            links = status.get('network_status')['links']
+            # get link status
+            for link, link_attributes in links.items():
+                self.links_state[link] = link_attributes.get('enabled')
```

Comment:
I believe link states should have booleans as values, so 
```suggestion
                self.links_state[link] = link_attributes.get('enabled', False)
```
to have a default value if `enabled` does not exist, or 
```suggestion
                self.links_state[link] = link_attributes['enabled']
```
to raise an error if `enabled` does not exist.